apps/spruce/src/pages/version/Metadata.tsx

```diff
@@ -152,6 +155,16 @@ export const Metadata: React.FC<Props> = ({ loading, version }) => {
           </InlineCode>
         </MetadataItem>
       )}
+      {isGithubMergePatch && (
+        <MetadataItem>
+          <StyledLink
```

Comment:
I think an external icon would be great 👍

Reply:
added! for reasons I don't understand, it doesn't show up on development environment, but it'll show up when deploying to staging/beta/etc